packages/cli/test/integration/credentials.api.test.ts

```diff
@@ -87,7 +87,7 @@ test('POST /credentials should fail with invalid inputs', async () => {
 
 test('POST /credentials should fail with missing encryption key', async () => {
 	const mock = jest.spyOn(UserSettings, 'getEncryptionKey');
-	mock.mockResolvedValue(undefined);
+	mock.mockRejectedValue(new Error('Encryption key not found.'));
```

Comment:
if we apply the change in `UserSettings.ts` then we could insert the variable here so even if we change the wording the tests wouldn't break

if we don't change it then we should move `Encryption key not found.` to a variable and import here 👍🏼 (also at the other places in this file)

Reply:
Exactly, the wording is not super important but def if we can get everything equal that is best for consistency